book/10-git-internals/sections/objects.asc

```diff
@@ -381,6 +381,14 @@ You can calculate the SHA-1 value of a string in Ruby by including the SHA1 dige
 => "bd9dbf5aae1a3862dd1526723246b20206e5fc37"
 ----
 
+Let's validate the hash with `git hash-object`. It shows that they are identical. Note that `echo` will add a trailing newline character in string, which is not we want in this case. `echo -n` can prevent from adding a trailing newline character.
```

Comment:
Could you reformat this so that each sentence is on its own line? It makes future PRs much easier to review.

Also, let's make it a bit more succinct. Something like this maybe?

```
Let's compare that to the output of `git hash-object`.
Here we use `echo -n` to prevent adding a newline to the input.
```

What do you think?